docs/standard/linq/project-object-graph.md

```diff
@@ -0,0 +1,543 @@
+---
+title: How to project an object graph - LINQ to XML
+description: You can use C# or Visual Basic to project or populate an object graph from XML. The example in this article shows how.
+ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 293d15d5-3eaf-48de-9a02-3e13cb117b5b
+---
+
+# How to project an object graph (LINQ to XML)
+
+You can use C# or Visual Basic to project or populate an object graph from XML.
+
+## Example: Populate an object graph from XML
+
+This example populates an object graph with the `Address`, `PurchaseOrder`, and `PurchaseOrderItem` classes from XML document [Sample XML file: Typical purchase order](sample-xml-file-typical-purchase-order.md).
+
+```csharp
+class Address
+{
+    public enum AddressUse
+    {
+        Shipping,
+        Billing,
+    }
+
+    private AddressUse addressType;
```

Comment:
I realize this is expanding scope, but is this a good time to remove the `private` fields and change the later properties to auto-implemented properties?

@mairaw

Reply:
I don't think this is my call. It certainly is beyond scope, because it is well beyond the attitude of "get the articles merged and go from there". That does necessarily argue against  it - we have already gone beyond scope, I think, which is not necessarily bad. 

I might or might not be the right person to make such changes. I don't really understand what you propose, but if it only requires knowledge of C# I would be fine. If it depends on knowing LINQ, I might not be the best person.

Reply:
Let's keep the task focused on bringing the articles over. And then you can make the improvements later @BillWagner?